tests/src/HTMLElement/elements/HTMLDocumentElement.js

```diff
@@ -0,0 +1,29 @@
+/*global test*/
+import assert from 'proclaim';
+import {readFileSync} from 'fs';
+
+const expect = assert.strictEqual;
+const throws = assert.throws;
+
+const lib = '../../../../lib/';
+
+const Document = require(lib + 'Document');
+const Doctype = require(lib + 'Doctype');
+
+const fileText = readFileSync(__dirname + '/../../../data/linkstest.html', { encoding: 'utf-8' });
```

Comment:
Well, you are right. I used that document, which were failing for me. I just moved tests from my application to html-document to reproduce problem, so I can find a problem. After I just left it as is.
I have some more plans on this document later (especially for easier testing of selectors etc.).
I can update tests if you think they are not clear now.